compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TaintAnalyzer.java

```diff
@@ -260,20 +250,25 @@ public static TaintAnalyzer getInstance(CompilerContext context) {
 
     public TaintAnalyzer(CompilerContext context) {
```

Comment:
This method seems to be a private method

Reply:
This is commonly `public` across all other compiler phases. Did not differ from the norm. Example: https://github.com/ballerina-platform/ballerina-lang/blob/450eb61585cce0765c2dd834fab54272653f52a1/compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/CodeAnalyzer.java#L233

Reply:
@sameerajayasoma @hasithaa is this something we can fix across all phases, or is there a reason for keeping the constructor `public`?